lua/lean/infoview.lua

```diff
@@ -503,7 +501,7 @@ function Pin:set_loading(loading)
   return false
 end
 
-Pin.update = a.void(function(self, force, delay, _, lean3_opts)
+function Pin:a_update(force, delay, _, lean3_opts)
```

Comment:
Can you name this something a bit more transparent? What is `a`? Is that short for async?